concent_api/core/management/commands/concent_adapt_data_to_current_version.py

```diff
@@ -0,0 +1,29 @@
+from logging import getLogger
+
+from django.core.management.base import BaseCommand
+from typing import Any
+
+from common.helpers import deserialize_message
+from core.models import StoredMessage
+from core.models import Subtask
+
+logger = getLogger(__name__)
+
+
+class Command(BaseCommand):
+    help = "Adapts whole database to current Concent's version"
```

Comment:
I would change it to `Removes Subtasks which related ReportComputedTask cannot be deserialized or its size field is not int.` or something more precise. 

`Adapts` may not make user aware that something might be deleted.

Reply:
Ok, so in future we will need to change this comment to adapt :P this comment to newer functions which will be provided by this script